### ISSUE TITLE
Change functionality and color of mission area dots on map

### ISSUE THREAD

**Issue by robynlloos** [uno-isqa-8950/uno-cpi#550]
- Change the colors of the dots to be more sophisticated and account for users with visual disabilities (do not use neon colors for the dots, or red and green because people who are color blind/have a visual disability cannot see these types of colors)
- Change the dots to be more transparent and include a darker outline around the dots
- Make the dots responsive - so smaller as you zoom into the map and larger as you zoom out so they are clear as the map changes - making all the dots are more visible
- Improve the look of the dots when there are multiple dots in the same location (see the original CPI map for example)

**Comment by caadler** [comment]
Maps have been converted to Google Maps and have been substantially updated.  New functionality will need to be logged in new tickets.
